x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/alerts/alerts_card_config.ts

```diff
@@ -0,0 +1,107 @@
+      }
+    ),
+    asset: {
+      type: CardSelectorListItemAssetType.video,
+      source: VIDEO_SOURCE,
+      alt: i18n.translate('xpack.securitySolution.onboarding.alertsCards.details.description', {
+        defaultMessage: 'Sort through alerts and drill down into its details',
+      }),
+    },
+  },
+  {
+    id: AlertsCardItemId.timeline,
+    title: i18n.translate('xpack.securitySolution.onboarding.alertsCards.timeline.title', {
+      defaultMessage: 'Investigate in Timeline',
+    }),
+    description: i18n.translate(
+      'xpack.securitySolution.onboarding.alertsCards.timeline.description',
+      {
+        defaultMessage: 'Streamline alert investigation with real-time visualization',
+      }
+    ),
+    asset: {
+      type: CardSelectorListItemAssetType.image,
+      source: alertTimelineImageSrc,
+      alt: i18n.translate('xpack.securitySolution.onboarding.alertsCards.timeline.description', {
+        defaultMessage: 'Streamline alert investigation with real-time visualization',
+      }),
+    },
+  },
+  {
+    id: AlertsCardItemId.analyzer,
+    title: i18n.translate('xpack.securitySolution.onboarding.alertsCards.analyzer.title', {
+      defaultMessage: 'Investigate in Analyzer',
+    }),
+    description: i18n.translate(
+      'xpack.securitySolution.onboarding.alertsCards.analyzer.description',
+      {
+        defaultMessage: 'Simplify alert analysis by visualizing threat detection processes',
+      }
+    ),
+    asset: {
+      type: CardSelectorListItemAssetType.video,
+      source: VIDEO_SOURCE,
+      alt: i18n.translate('xpack.securitySolution.onboarding.alertsCards.analyzer.description', {
+        defaultMessage: 'Simplify alert analysis by visualizing threat detection processes',
+      }),
+    },
+  },
+  {
+    id: AlertsCardItemId.sessionView,
+    title: i18n.translate('xpack.securitySolution.onboarding.alertsCards.sessionView.title', {
+      defaultMessage: 'Investigate in Session View',
+    }),
+    description: i18n.translate(
+      'xpack.securitySolution.onboarding.alertsCards.sessionView.description',
+      {
+        defaultMessage: 'Centralized threat analysis and response with real-time data insights',
+      }
+    ),
+    asset: {
+      type: CardSelectorListItemAssetType.image,
+      source: sessionViewImageSrc,
+      alt: i18n.translate('xpack.securitySolution.onboarding.alertsCards.sessionView.description', {
+        defaultMessage: 'Centralized threat analysis and response with real-time data insights',
+      }),
+    },
+  },
+];
+
+export const ALERTS_CARD_ITEMS_BY_ID = Object.fromEntries(
+  ALERTS_CARD_ITEMS.map((card) => [card.id, card])
+) as Record<AlertsCardSelectorListItem['id'], AlertsCardSelectorListItem>;
+
+export const ALERTS_CARD_SELECTOR_ITEMS = ALERTS_CARD_ITEMS.map<CardSelectorListItem>(
+  (alertsItem) => ({
+    id: alertsItem.id,
+    title: alertsItem.title,
+    description: alertsItem.description,
+  })
+);
```

Comment:
Is TS complaining if we use `ALERTS_CARD_ITEMS` array in the component directly?

Reply:
Yes, because the `CardSelectorList` component is waiting the `CardSelectorListItem` type that does not includes `asset` prop. 
To improve this I've made some changes on the `CardSelectorList` to accept more properties in a more agnostic way.  So with this changes I can remove ` ALERTS_CARD_SELECTOR_ITEMS` mapping and use `ALERTS_CARD_ITEMS` instead.

```
export interface CardSelectorListProps<T extends CardSelectorListItem> {
  items: T[];
  onSelect: (item: T) => void;
  selectedItem: T;
  title?: string;
}

Reply:
I don't think that's necessary, using generics is a smart idea to solve this kind of problem, but in this case TS does not complain. That's why I first asked if you checked what is the TS error, because there's no TS error to fix. 

Explanation: the `CardSelectorAssetListItem` type definition already extends `CardSelectorListItem`, so TS understands that it has all the properties it needs. 

There's no problem defining:
```
export interface CardSelectorListProps {
  items: CardSelectorListItem[];
  onSelect: (item: CardSelectorListItem) => void;
  selectedItem: CardSelectorListItem;
  title?: string;
}
```

and passing `items` defined as `CardSelectorAssetListItem` to the component.

Reply:
OMG you are right! It's not necessary so I've removed the generic types :) 
I really appreciate your feedback 👍